README.md

```diff
@@ -61,6 +61,43 @@ PyPI, which is recommended to restrict the access the action has.
 The secret used in `${{ secrets.PYPI_API_TOKEN }}` needs to be created on the
 settings page of your project on GitHub. See [Creating & using secrets].
 
+### Publishing with OpenID Connect
+
+**IMPORTANT**: This functionality is in beta, and will not work for you
+unless you're a member of the PyPI OIDC beta testers' group. For more
+information, see [warehouse#12965].
+
+This action supports PyPI's [OpenID Connect publishing]
+implementation, which allows authentication to PyPI without a manually
+configured API token or username/password combination. To perform
+[OIDC publishing](OpenID Connect Publishing) with this action, your project's
```

Comment:
This would link to `https://OpenID Connect Publishing`. The correct syntax is:
```suggestion
[OIDC publishing][OpenID Connect Publishing] with this action, your project's
```

Reply:
Thanks!